packages/serverless/src/apollo/app/database.ts

```diff
@@ -183,12 +184,23 @@ const dbChangeOwner = authorizeWrapper(
     changeDatabaseOwner(args.databaseName, ctx.userName, args.newOwner)
 );
 
+const dbExist = publicWrapper(
+  Joi.object({
+    databaseName,
+  }),
+  async (_root: unknown, args: TDatabaseRequest, _ctx) => {
+    const { databases } = await DB.service.client.db().admin().listDatabases();
+    return databases.some((db) => db.name === args.databaseName);
```

Comment:
Why not?
```
return databases.indexOf(args.databaseName) >= 0;
```

Reply:
I would recommend to check if setting exist for the database, if so returns true